src/Compilers/CSharp/Test/Symbol/Compilation/ReferenceManagerTests.cs

```diff
@@ -187,15 +168,7 @@ public void F()
                    new CSharpCompilationReference(refLibV2)
                });
 
-            // TODO (tomat): we should display paths rather than names "RefLibV2" and "Lib"
-
-            main13.VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=19&length=17)

According to the comment above, it looks like the goal of the test was to verify that: "higher version should be preferred over lower version regardless of the order of the references". It also looks like the fact was verified indirectly by checking diagnostics. If we no longer can rely on this approach, we should find a different way to verify the outcome. It feels like we might want other tests that primarily were verifying presence of the warnings (at least all tests in this file) be adjusted in a similar way.